### PR TITLE
MC-19390: Fix API docs

### PR DESCRIPTION
**Pull request by remcalu**
#### Changes made

- Clarify various parameters
- Fix typos

#### Related PRs
- [PR#494](https://github.com/cloudops/cloudmc-gcp-plugin/pull/494)

### CODE REVIEW

File: source/includes/gcp/_firewall_rules.md [hunk 1]
```diff
@@ -153,41 +153,51 @@ Retrieve a firewall rules in a given [environment](#administration-environments)
...
-`direction`<br/>*string* | Direction of traffic to which this firewall applies, either INGRESS or EGRESS. The default is INGRESS.
-`priority`<br/>*string* | Priority for this rule. This is an integer between 0 and 65535, both inclusive. The default value is 1000.
+`network`<br/>*string* | URL of the network resource for this firewall rule.
 `range`<br/>*Array[string]* | The source or destination range depending on the `direction` specified. The firewall rule applies only to traffic that has a source/destination IP address in these ranges. These ranges must be expressed in CIDR format.
-
+`all`<br/>*boolean* | Specifies if the firewall rule should allow all, or some protocols and ports. This parameter overwrites the `tcpPorts`, `udpPorts` and `protocols` parameters.
+`tcpPorts`<br/>*Array[string]* | The tcp ports on which to apply the rule. These must be in the range [0, 65535).
+`udpPorts`<br/>*Array[string]* | The udp ports on which to apply the rule. These must be in the range [0, 65535).
+`protocols`<br/>*Array[string]* | Supported protocols are: icmp, esp, ah, sctp, ipip or a valid [decimal IP protocol number].
```
Comment (lamminade) [inline source/includes/gcp/_firewall_rules.md]:
```suggestion
`protocols`<br/>*Array[string]* | Supported protocols are: icmp, esp, ah, sctp, ipip or a valid [decimal IP protocol number](https://www.iana.org/assignments/protocol-numbers/protocol-numbers.xhtml).
```
forgot the link here

**Review by patrickspensieri — APPROVED** [review]
Looks good to me 👍

File: source/includes/gcp/_firewall_rules.md [hunk 2]
```diff
@@ -153,41 +153,51 @@ Retrieve a firewall rules in a given [environment](#administration-environments)
...
 
 Required | &nbsp;
 ------- | -----------
-`action`<br/>*string* | The firewall rule type: 'allow' (allow traffic) or 'deny' (deny traffic).
-`direction`<br/>*string* | Direction of traffic to which this firewall applies, either INGRESS or EGRESS. The default is INGRESS.
-`priority`<br/>*string* | Priority for this rule. This is an integer between 0 and 65535, both inclusive. The default value is 1000.
+`network`<br/>*string* | URL of the network resource for this firewall rule.
 `range`<br/>*Array[string]* | The source or destination range depending on the `direction` specified. The firewall rule applies only to traffic that has a source/destination IP address in these ranges. These ranges must be expressed in CIDR format.
-
+`all`<br/>*boolean* | Specifies if the firewall rule should allow all, or some protocols and ports. This parameter overwrites the `tcpPorts`, `udpPorts` and `protocols` parameters.
```
Comment (lamminade) [inline source/includes/gcp/_firewall_rules.md]:
```suggestion
`all`<br/>*boolean* | Specifies if the firewall rule should allow all or some protocols and ports. This parameter overwrites the `tcpPorts`, `udpPorts` and `protocols` parameters.
```

File: source/includes/gcp/_firewall_rules.md [hunk 3]
```diff
@@ -153,41 +153,51 @@ Retrieve a firewall rules in a given [environment](#administration-environments)
...
 `name`<br/>*string* | The display name of the firewall rule. A default name will be created if there isn't one provided.
-`all`<br/>*boolean* | Specifies if the firewall rule is an allow all or deny all rule.
-`tcpPorts`<br/>*Array[string]* | The tcp ports on which to apply the rule. These must be in the range [0, 65535).
-`udpPorts`<br/>*Array[string]* | The udp ports on which to apply the rule. These must be in the range [0, 65535).
-`protocols`<br/>*Array[string]* | Supported protocols are: icmp, esp, ah, sctp, ipip or a valid [decimal IP protocol number](https://www.iana.org/assignments/protocol-numbers/protocol-numbers.xhtml).
-
+`action`<br/>*string* | The firewall rule type: 'allow' (allow traffic) or 'deny' (deny traffic). The default value is 'allow'.
+`priority`<br/>*string* | Priority for this rule. This is an integer between 0 and 65535, both inclusive. The default value is 1000.
+`direction`<br/>*string* | Direction of traffic to which this firewall applies, either INGRESS or EGRESS. The default is INGRESS.
+(https://www.iana.org/assignments/protocol-numbers/protocol-numbers.xhtml).
```
Comment (lamminade) [inline source/includes/gcp/_firewall_rules.md]: found the missing link :p
```suggestion
```

**Review by lamminade — CHANGES_REQUESTED** [review]
couple quick fixes and one suggestion for clarity

File: source/includes/gcp/_firewall_rules.md (hunk 1 above)
Comment (remcalu) [reply source/includes/gcp/_firewall_rules.md]: Oops, nice catch 👌🏻

File: source/includes/gcp/_firewall_rules.md (hunk 3 above)
Comment (remcalu) [reply source/includes/gcp/_firewall_rules.md]: 😇

**Review by lamminade — APPROVED** [review]

**Pull request merged**